src/Docker/Runner/Authorization.php

```diff
@@ -22,7 +22,6 @@ public function __construct(
         $this->componentId = $componentId;
         $this->encryptor = $encryptor;
         $this->oauthClientV3 = $clientV3;
-        $this->oauthClientV3->enableReturnArrays(true);
```

Comment:
zrušeno, už to vrací jen pole https://github.com/keboola/oauth-v2-php-client/pull/11#discussion_r978113519